freeipa.spec.in

```diff
@@ -125,7 +125,7 @@ BuildRequires:  python-lxml
 # 5.0.0: QRCode.print_ascii
 BuildRequires:  python-qrcode-core >= 5.0.0
 # 1.11.0: resolver.YXDOMAIN, Resolver.set_flags
```

Comment:
Sorry, I didn't see the comment before. Please update or remove the outdated comment.